Riot/Modules/Room/VoiceMessages/VoiceMessageAudioRecorder.swift

```diff
@@ -0,0 +1,115 @@
+// 
+// Copyright 2021 New Vector Ltd
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+import Foundation
+import AVFoundation
+
+private let silenceThreshold: Float = -50.0
```

Comment:
Can we use this constant internally in `VoiceMessageAudioRecorder`? We are used to group constant in a class like this:
```
private enum Constants {
static let silenceThreshold: Float = -50.0
}
```
It's possible to rename the enum name or create severals. enum are used as struct. The detail is enum do not have initializer.